packages/components/src/Stepper/Stepper.component.js

```diff
@@ -170,25 +172,28 @@ function Stepper({ steps, title, renderActions, children }) {
 		}
 	}, [steps]);
 
+	if (isInError) {
+		const errorStep = getStepInError(steps);
```

Comment:
Can it be possible that we get 'undefined' here, let's say there's no status LOADING_STEP_STATUSES.FAILURE ??